src/Services/Air/Air.js

```diff
@@ -350,7 +350,10 @@ module.exports = (settings) => {
 
       return this.getUniversalRecordByPNR(options)
         .then((ur) => {
-          const record = Array.isArray(ur) ? ur[0] : ur;
+          ur = Array.isArray(ur) ? ur[0] : ur;
```

Comment:
Unneeded mutation

Reply:
@dchertousov, could you explain your observation ?

I was just keen on getting this to work, since I am already deploying the wrapper on AWS Serverless using **sam** and **cloudformation** . After the change, the cancellation is working smoothly.

Reply:
I think @dchertousov means you can avoid mutating the parameter, instead declare a new variable.